pkg/sql/compile/scope.go

```diff
@@ -101,6 +110,22 @@ func (s *Scope) MergeRun(c *Compile) error {
 				}()
 				err = cs.ParallelRun(c)
 			}(s.PreScopes[i])
+		case External:
```

Comment:
All these *2 stuff, out of control.   Why do we need that, can we fix the *1 instead?

Reply:
Why is External a type, and isn't External supposed to be a specialization of ValueScan?